go/src/koding/db/models/groupdata.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+// GroupData holds a group's private info.
+type GroupData struct {
+	ID   string `json:"_id"`
+	Slug string `json:"slug"`
+	Data *Data  `bson:"data,omitempty" json:"data,omitempty"`
```

Comment:
1) What advantages we are going to have when we dont have a pointer?
2) if bson tags are not defined, lowercased version of the fields are used, so we dont have to define. but i've realized i used string in id field, need to change to bson.ObjectId